region/brinstar/pink/Pink Brinstar Power Bomb Room.json

```
@@ -71,6 +71,17 @@
       "obstacleType": "abstract"
     }
   ],
+  "reusableRoomwideNotable": [
+    {
+      "name": "Mission Impossible Shinespark",
+      "note": [
+        "Gain a shinecharge by running right-to-left, either using the short runway (with unbroken Super block) or an extended runway by taking a spike hit.",
+        "Then spin jump, hit the ceiling, aim down, break the Super block, and turn around to quick drop through the crumble block.",
```

```suggestion
        "Then spin jump for some height but not too much, aim down, break the Super block, and turn around to quick drop through the crumble block.",
```
I think if you jump too high then you can run out of shinecharge frames